src/platform/silabs/CHIPPlatformConfig.h

```diff
@@ -127,6 +127,10 @@ static_assert(CHIP_CONFIG_ICD_OBSERVERS_POOL_SIZE >= 3, "ICD Observers pool size
 
 #endif // defined(CHIP_CONFIG_ENABLE_ICD_SERVER) && CHIP_CONFIG_ENABLE_ICD_SERVER
 
+#ifndef CHIP_CONFIG_MRP_RETRY_INTERVAL_SENDER_BOOST
+#define CHIP_CONFIG_MRP_RETRY_INTERVAL_SENDER_BOOST (2500_ms32)
```

Comment:
are we not giving time unit in these MACROs ?

Reply:
The macro forces the time to be part of the configuration value `_ms32` is a time util macro that informs that is a `Milliseconds32` wihch means milliseconds in a uint32.